File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    num_list = []
    answer = []
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                num_list.append(nums[i] + nums[j] + nums[k])
    for i in num_list:
        if all(i % j for j in range(2, i)):
            answer.append(i)
    return len(answer)

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.java
import java.util.*;

class Solution {
    public int solution(int[] nums) {
        int answer = 0;
        List<Integer> lst = new ArrayList<>();
        
        for(int i=0; i<nums.length; i++) {
            for(int j=i+1; j<nums.length; j++) {
                for(int k=j+1; k<nums.length; k++) {
                    lst.add(nums[i] + nums[j] + nums[k])  ;          
                }
            }
        }
        
        for(int i : lst) {
            if(isPrime(i))
                answer++;
        }
        
        return answer;
    }
    
    public boolean isPrime(int num) {
        for(int i=2; i<num; i++){
            if(num%i==0)
                return false;
        }
        return true;
    }
}